### PR TITLE
Add Graphics to the Match3 env

### PR DESCRIPTION
**Pull request by Hunter-Unity**
### Proposed change(s)
Adds Graphics to the M3 env. Spawns gameobject "tiles" in game mode. Added image to the Match3 docs.

### Types of change(s)

- [ ] Bug fix
- [x] New feature
- [ ] Code refactor
- [ ] Breaking change
- [x] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: Project/Assets/ML-Agents/Examples/Match3/Scripts/Match3Drawer.cs [hunk 1]
```diff
@@ -9,37 +10,108 @@ public class Match3Drawer : MonoBehaviour
...
+            }
+        }
 
         void OnDrawGizmos()
         {
             // TODO replace Gizmos for drawing the game state with proper GameObjects and animations.
             var cubeSize = .5f;
-            var cubeSpacing = .75f;
-            var matchedWireframeSize = .5f * (cubeSize + cubeSpacing);
+            var matchedWireframeSize = .5f * (cubeSize + CubeSpacing);
```
Comment (chriselion) [inline Project/Assets/ML-Agents/Examples/Match3/Scripts/Match3Drawer.cs:97]: Do we even need the gizmos code anymore?

Comment (chriselion) [reply Project/Assets/ML-Agents/Examples/Match3/Scripts/Match3Drawer.cs:97]: Even if we want to keep it around, you can remove the TODO above :)

**Review by chriselion — APPROVED** [review]
Looks good.

Is there a reason that there's both
Project/Assets/ML-Agents/Examples/Match3/Prefabs/FBX/Match.fbx
Project/Assets/ML-Agents/Examples/Match3/Prefabs/FBX/3.fbx
? Did "Match 3" get turned into two files?

File: Project/Assets/ML-Agents/Examples/Match3/Scripts/Match3Drawer.cs (hunk 1 above)
Comment (Hunter-Unity) [reply Project/Assets/ML-Agents/Examples/Match3/Scripts/Match3Drawer.cs:96]: changed: removed the todo. also, the gizmos still show some of the debug features you made. there's still value there so i'd say we can leave for now

**Comment by Hunter-Unity** [comment]
> Looks good.
> 
> Is there a reason that there's both
> Project/Assets/ML-Agents/Examples/Match3/Prefabs/FBX/Match.fbx
> Project/Assets/ML-Agents/Examples/Match3/Prefabs/FBX/3.fbx
> ? Did "Match 3" get turned into two files?

i brought them in as 3 separate meshes so i could independently scale the 3 so this was intended

**Pull request merged**